example-dvc-experiments/generate.bash

```diff
@@ -87,37 +64,33 @@ git tag "git-init"
 
 cp -r "${HERE}"/code/src .
 cp "${HERE}"/code/requirements.txt .
+cp "${HERE}"/code/requirements-macos.txt .
 cp "${HERE}"/code/params.yaml .
-pip install -r "${REPO_PATH}"/requirements.txt
+if [[ $(uname -s) == 'Darwin' ]] ; then
+    pip install -r "${REPO_PATH}"/requirements-macos.txt
```

Comment:
It looks macOS requires the whole generation to depend on [conda](https://developer.apple.com/metal/tensorflow-plugin/) in M1 macs. This can be solved, but it requires a `generate-macos.bash` that will use `conda` instead of `pip`. I can work on this, WDYT? @shcheklein

Reply:
Keeping this as is for future, maybe we'll be able to install TF with `pip` on macs someday.